src/data/finset/basic.lean

```diff
@@ -1895,6 +1895,11 @@ by { subst h, simp }
 theorem map_map {g : β ↪ γ} : (s.map f).map g = s.map (f.trans g) :=
 eq_of_veq $ by simp only [map_val, multiset.map_map]; refl
 
+lemma map_comm {β'} {f : β ↪ γ} {g : α ↪ β} {f' : α ↪ β'} {g' : β' ↪ γ}
```

Comment:
This lemma doesn't look like a commutativity lemma to me,  there are too many arguments!


Can you point to another lemma named in this way?